graph4nlp/pytorch/modules/graph_embedding_initialization/embedding_construction.py

```diff
@@ -140,6 +140,9 @@ def __init__(
             "w2v_bert",
             "w2v_bert_bilstm",
             "w2v_bert_bigru",
+            "w2v_amr",
```

Comment:
Why amr shoud be a general embedding strategy?
I don't think it should be regarded as a new general embedding strategy.